src/components/DateRangeFIlter/DateRangeFilter.tsx

```diff
@@ -15,11 +15,16 @@ import { RadioGroup, RadioGroupItem } from '@/components/ui/radio-group.tsx';
 import { cn } from '@/lib/utils.ts';
 import { Route } from '@/routes/(admin)/_admin.access-keys.tsx';
 
-export const DateRangeFilter = () => {
+export interface DateRangeFilterProps {
+  withRevokedRange?: boolean;
+}
+
+export const DateRangeFilter = ({ withRevokedRange }: DateRangeFilterProps) => {
   const [dateRangeOption, setDateRangeOption] = useState<string>('createdRange');
+  const currentDate = new Date();
   const [date, setDate] = React.useState<DateRange | undefined>({
```

Comment:
Is the `undefined` part necessary for `const [date, setDate] = React.useState<DateRange | undefined>` ?

Reply:
Yes, it is. <Calendar onSelect> need type <DateRange | undefined>